src/installer/pkg/packaging/snaps/dotnet-runtime-3.1/snap/snapcraft.yaml

```diff
@@ -0,0 +1,45 @@
+name: dotnet-runtime-31
+base: core18
+version: 3.1.0
```

Comment:
Why are we keeping the snap files for older / already shipped versions in master?

Reply:
@dagood , what would you recommend we do with these snap files as we moved them from core-setup to runtime as per your suggestion? 

Ideally, we would want to keep all of the snap files in one place (one storage location for all) to not litter them across branches as per product release.

Reply:
These were put in Core-Setup master a while back just to have a place to check them in, I don't think they have a good home. Ideally they would live near the artifacts they actually use, Core-Setup 3.0/3.1 and Runtime master for the ones that ship the runtime, plus I think Core-SDK 3.0/3.1 and SDK master for the one shipping SDKs. They would also ideally actually be used during the build and produce the snaps as build artifacts. https://github.com/dotnet/core-setup/issues/4789 tracks the overall effort.

The suggestion to move the PR to dotnet/runtime isn't because it belongs here, it's solely because core-setup is now consolidated into this repo. (Core-Setup no longer has a master branch.)

Reply:
Context, the Core-Setup PR: https://github.com/dotnet/core-setup/pull/8912

Reply:
The snap files look very similar to docker files. Would it make sense to keep the snap and docker files together?

Reply:
As long as it is a consistent home (that is not spread across repos) for these snap files it should be fine to keep them in any logical repo as they are independent of anything else. 
Yes, they do look similar to docker files but I am not sure if they fit right in the repo that is specifically created for docker: https://github.com/dotnet/**dotnet-docker**  

Cc: @MichaelSimons

Reply:
Adding the snap artifacts to the docker repo would go against the source code repo patterns used within the Docker ecosystem.  The Docker practice is to keep all Dockerfiles in a standalone github repo.  I would not be in favor of putting the snap source in the Docker repo.